compose/ui/ui-text/src/skikoMain/kotlin/androidx/compose/ui/text/platform/PlatformFont.skiko.kt

```diff
@@ -320,3 +351,18 @@ private val GenericFontFamiliesMapping: Map<String, List<String>> by lazy {
             )
     }
 }
+
+internal fun FontVariation.Settings.toSikaFontVariationList(): List<org.jetbrains.skia.FontVariation> {
+    return settings.map { setting ->
+        org.jetbrains.skia.FontVariation(setting.axisName, setting.toVariationValue(null))
+    }
+}
+
+/**
+ * Bind the font variation settings to the Skia typeface.
+ */
+internal fun SkTypeface.bindVariantSettings(variationSettings: FontVariation.Settings): SkTypeface {
```

Comment:
Since it clones the original font, I would name it `cloneWithVariationSettings`

Reply:
Updated function name and coment